services/core-web/cypress/e2e/majorprojects.cy.ts

```diff
@@ -0,0 +1,86 @@
+describe("Major Projects", () => {
+    beforeEach(() => {
+        cy.login();
+        cy.wait(5000);
```

Comment:
I'm wondering if it's possible to set it to `wait for up to 5 seconds, until the Major Projects button appears`, instead of `wait for 5 seconds, and then look for the Major Projects button`.

Found some suggestions in their documentation: https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
You can set the universal timeout length in the the cypress config, or you can always add a timeout to any `cy.get()` as well.  ie. `cy.get('table[style="table-layout: auto;"]', {timeout: 5000})`